clang/include/clang/Driver/Options.td

```diff
@@ -1822,6 +1822,9 @@ def fstrict_vtable_pointers: Flag<["-"], "fstrict-vtable-pointers">,
 def fstrict_overflow : Flag<["-"], "fstrict-overflow">, Group<f_Group>;
 def fintelfpga : Flag<["-"], "fintelfpga">, Group<f_Group>,
   Flags<[CC1Option, CoreOption]>, HelpText<"Perform ahead of time compilation for FPGA">;
+def fsycl_enable_usm_address_spaces : Flag<["-"], "fsycl-enable-usm-address-spaces">,
+ Group<f_Group>, Flags<[CC1Option, CoreOption]>,
+ HelpText<"Add support for SPV_INTEL_usm_storage_classes feature">;
```

Comment:
Is there a better description to put in the help?  Maybe just `Enable usm address spaces`

Reply:
I rephrased it to: "Enable SPV_INTEL_usm_storage_classes extension". Will it work?
Basically, this option will be deleted in a close future - for FPGA generation of new storage classes will be enabled by default, while for other targets it will be disabled.

Reply:
As it is temporary, I'm OK with it.  Just not a fan of exposing what looks to be internal nomenclature.